### PR TITLE
Update assign-role.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a note per users request to reduce ambiguity

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/memdocs/issues/2719

**Comment by PRMerger19** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: memdocs/intune/fundamentals/assign-role.md [hunk 1]
```diff
@@ -55,6 +55,9 @@ To create, edit, or assign roles, your account must have one of the following pe
 
 8. On the **Review + Create** page, when you're done, choose **Create**. The new assignment is displayed in the list of assignments.
 
+    > [!NOTE] 
+    > You create scope groups and when you assign a scope tag, you can only target groups that are listed in the Scope (Groups) of your role assignment.
```
Comment (JohanFreelancer9) [inline memdocs/intune/fundamentals/assign-role.md:59]:
```suggestion
    > When you create scope groups and assign a scope tag, you can only target groups that are listed in the Scope (Groups) of your role assignment.
```

**Review by JohanFreelancer9 — COMMENTED** [review]
Copy review.

**Comment by PRMerger7** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by Smritib17 — APPROVED** [review]

**Pull request merged**